Templating/Helper/NumberHelper.php

```diff
@@ -119,63 +158,115 @@ public function formatCurrency($number, $currency, array $attributes = array(),
      *
-     * @return string
+     * @return string The formatted number.
      */
-    public function formatOrdinal($number, array $attributes = array(), array $textAttributes = array(), $locale = null)
+    public function formatOrdinal($number, array $attributes = array(), array $textAttributes = array(), $symbols = array(), $locale = null)
     {
-        return $this->format($number, \NumberFormatter::ORDINAL, $attributes, $textAttributes, $locale);
+        list($locale, $symbols) = $this->normalizeMethodSignature($symbols, $locale);
+
+        return $this->format($number, \NumberFormatter::ORDINAL, $attributes, $textAttributes, $symbols, $locale);
     }
 
     /**
-     * format a value with its provided style.
+     * Formats a number according to the specified locale and \NumberFormatter
+     * attributes.
      *
-     * @param float $number
-     * @param int style
-     * @param array $attributes
-     * @param array $textAttributes
-     * @param null  $locale
+     * @param string|float|int $number         The number to format.
+     * @param                  $style
+     * @param array            $attributes     The attributes used by the formatter.
+     * @param array            $textAttributes The text attributes used by the formatter.
+     * @param array            $symbols        The symbols used by the formatter.
+     * @param string|null      $locale         The locale used to format the number.
      *
      * @return string
      */
-    public function format($number, $style, array $attributes = array(), array $textAttributes = array(), $locale = null)
+    public function format($number, $style, array $attributes = array(), array $textAttributes = array(), $symbols = array(), $locale = null)
     {
-        $formatter = $this->getFormatter($locale ?: $this->localeDetector->getLocale(), $style, $attributes, $textAttributes);
+        list($locale, $symbols) = $this->normalizeMethodSignature($symbols, $locale);
+
+        $formatter = $this->getFormatter($locale ?: $this->localeDetector->getLocale(), $style, $attributes, $textAttributes, $symbols);
 
         return $this->fixCharset($formatter->format($number));
     }
 
     /**
-     * @param $culture
-     * @param $style
-     * @param array $attributes
-     * @param array $textAttributes
+     * Normalizes the given arguments according to the new function signature.
+     * It asserts if neither the new nor old signature matches. This function
+     * is public just to prevent code duplication inside the Twig Extension.
+     *
+     * @param array  $symbols The symbols used by the formatter.
+     * @param string $locale  The locale
+     *
+     * @return array Arguments list normalized to the new method signature.
+     *
+     * @throws \BadMethodCallException If the arguments does not match any signature.
+     *
+     * @internal
+     */
+    public function normalizeMethodSignature($symbols, $locale)
```

Comment:
Docblock is already fixed, but I don't know why it still shows the old code here.